bundle/jsky.app.ot/src/main/java/jsky/app/ot/gemini/editor/targetComponent/TelescopePosTableWidget.java

```diff
@@ -290,20 +290,13 @@ private static Icon bagsIcon(boolean enabled) {
                     final GuideProbe guideProbe = gt.getGuider();
                     final boolean isActive = ctx.exists(c -> GuideProbeUtil.instance.isAvailable(c, guideProbe));
                     final Option<SPTarget> primary = gt.getPrimary();
-                    final Option<SPTarget> bagsTarget = gt.getBagsResult().targetAsJava();
```

Comment:
I think we're going to make a completely separate branch for BAGS to avoid having to manually rebag everything, so I don't think it will matter too much.